utils/debug/3dTilesDebug.js

```diff
@@ -1,88 +1,39 @@
-import View from 'Core/View';
-import GeometryLayer from 'Layer/GeometryLayer';
-import { C3DTilesBoundingVolumeTypes } from 'Core/3DTiles/C3DTilesEnums';
+import { DebugTilesPlugin } from '3d-tiles-renderer';
+
 import { PNTS_MODE, PNTS_SHAPE, PNTS_SIZE_MODE } from 'Renderer/PointsMaterial';
 import GeometryDebug from './GeometryDebug';
 
-const bboxMesh = new THREE.Mesh();
-
 export default function create3dTilesDebugUI(datDebugTool, view, _3dTileslayer) {
     const gui = GeometryDebug.createGeometryDebugUI(datDebugTool, view, _3dTileslayer);
 
     // add wireframe
     GeometryDebug.addWireFrameCheckbox(gui, view, _3dTileslayer);
 
-    // Bounding box control
-    const boundingVolumeID = `${_3dTileslayer.id}_bounding_volume_debug`;
-
-    function debugIdUpdate(context, layer, node) {
-        // Tile (https://github.com/CesiumGS/3d-tiles/blob/main/specification/schema/tile.schema.json) containing
-        // metadata for the tile
-        const tile = node.userData.metadata;
-
-        // Get helper from the node if it has already been computed
-        let helper = node.userData.boundingVolumeHelper;
-
-        // Hide bounding volumes if 3D Tiles layer is hidden
-        if (helper) {
-            helper.visible = !!(layer.visible && node.visible);
-            return;
-        }
-
-        if (layer.visible && node.visible && tile.boundingVolume) {
-            if (tile.boundingVolume.initialVolumeType === C3DTilesBoundingVolumeTypes.box) {
-                bboxMesh.geometry.boundingBox = tile.boundingVolume.volume;
-                helper = new THREE.BoxHelper(bboxMesh);
-                helper.material.linewidth = 2;
-                // compensate GLTF orientation correction based on gltfUpAxis only for b3dm tiles
-                if (tile.content?.uri && tile.content?.uri.endsWith('b3dm')) {
-                    const gltfUpAxis = _3dTileslayer.tileset.asset.gltfUpAxis;
-                    if (gltfUpAxis === undefined || gltfUpAxis === 'Y') {
-                        helper.rotation.x = -Math.PI * 0.5;
-                    } else if (gltfUpAxis === 'X') {
-                        helper.rotation.z = -Math.PI * 0.5;
-                    }
-                    helper.updateMatrix();
-                }
-            } else if (tile.boundingVolume.initialVolumeType === C3DTilesBoundingVolumeTypes.sphere ||
-                       tile.boundingVolume.initialVolumeType === C3DTilesBoundingVolumeTypes.region) {
-                const geometry = new THREE.SphereGeometry(tile.boundingVolume.volume.radius, 32, 32);
-                const material = new THREE.MeshBasicMaterial({ wireframe: true, color: Math.random() * 0xffffff });
-                helper = new THREE.Mesh(geometry, material);
-            } else {
-                console.warn(`[3D Tiles Debug]: Unknown bounding volume: ${tile.boundingVolume}`);
-                return;
-            }
+    gui.add({ frozen: _3dTileslayer.frozen }, 'frozen').onChange(((value) => {
```

Comment:
Removed it.

Reply:
Ok, sorry, I thought it wouldn't work with `3d-tiles-renderer` but it will since it is handled in the`MainLoop` and it will prevent to call `OGC3DTilesLayer.preUpdate`